phoenix-shared/src/commonMain/sqldelight/channelsdb/fr/acinq/phoenix/db/sqldelight/ChannelsDatabase.sq

```diff
@@ -35,7 +35,7 @@ closeLocalChannel:
 UPDATE local_channels SET is_closed=1 WHERE channel_id=?;
 
 listLocalChannels:
-SELECT data FROM local_channels WHERE is_closed=0;
+SELECT channel_id, data FROM local_channels WHERE is_closed=0;
```

Comment:
Adding the `channel_id` here allows us to reuse in `listLocalChannels` the `mapChannelData` already used by `getChannel`. Also, lets us log the id of the faulty channel.